pkg/plugins/policies/meshratelimit/plugin/v1alpha1/plugin.go

```diff
@@ -11,6 +11,7 @@ import (
 	core_xds "github.com/kumahq/kuma/pkg/core/xds"
 	"github.com/kumahq/kuma/pkg/plugins/policies/core/matchers"
 	core_rules "github.com/kumahq/kuma/pkg/plugins/policies/core/rules"
+	inbound2 "github.com/kumahq/kuma/pkg/plugins/policies/core/rules/inbound"
```

Comment:
I would name it better, maybe `rules_inbound`?

Reply:
renamed